### PR TITLE
Adding footer to NTT

### LINKED ISSUE #60: Add footer with dynamic version tags
The NTT currently lacks a footer - we should add one similar to the DD with links and a dynamic version tag

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #60 

Similar to transitmatters/t-performance-dash#68

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
@idreyn @austinjpaul @mathcolo haven't gotten a chance to test this yet due to some postgres issues but feel free to comment on the initial code if you have any thoughts

**Comment by mathcolo** [comment]
@nathan-weinberg This is dandy dandy!! Could you add a screenshot of what it looks like?

File: server/mbta_api.py [hunk 1]
```diff
@@ -161,7 +166,9 @@ async def initial_request_data(route_ids, test_mode=False):
         ]
     )
     sightings = get_recent_sightings_for_lines()
+    git_tag = get_git_tag()
     return {
+        "version": git_tag,
```
Comment (mathcolo) [inline server/mbta_api.py:171]: I like how clean this is tbh

**Comment by nathan-weinberg** [comment]
> @nathan-weinberg This is dandy dandy!! Could you add a screenshot of what it looks like?

I would but still not working unfortunately :laughing: The correct value is betting captured and passed to the frontend, but for some reason the footer is not actually visible in the page - maybe something is amiss with the CSS?

File: src/main.css [hunk 2]
```diff
@@ -283,3 +283,9 @@ body {
     padding: 2px 4px;
     border-radius: 2px;
 }
+
+.footer {
+    position: fixed;
+    top: 0;
```
Comment (idreyn) [inline src/main.css:289]: I think you want `bottom` but not `top` here...and maybe `left: 0` too?

**Pull request merged**